.github/workflows/windows.yml

```diff
@@ -79,15 +79,16 @@ jobs:
           path: _build/install.log
 
       - name: check
-        continue-on-error: true
```

Comment:
What about moving this to two separate steps? One "check" with `continue-on-error` and another `test` that should never fail? After all these run as separate checks for win32 in any case.